src/js/text/main.html

```diff
@@ -10,11 +10,31 @@
         </div>
         <div id="ytGuPlayer"></div>
     </div>
-
+    <div class="docs--about-wrapper">
+      <div class="docs--about-body">
+        <p class="docs--about-headline">About the Guardian Bertha documentary partnership​</p>
```

Comment:
this could be possibly moved out into an HTML import and then imported into JS (see example in README). I am not wedded to this, it does provide some separation.